x/leverage/keeper/collateral.go

```diff
@@ -89,14 +88,14 @@ func (k Keeper) setCollateralAmount(ctx sdk.Context, borrowerAddr sdk.AccAddress
 
 // GetTotalCollateral returns an sdk.Coin representing how much of a given uToken
 // the x/leverage module account currently holds as collateral. Non-uTokens return zero.
-func (k Keeper) GetTotalCollateral(ctx sdk.Context, denom string) sdkmath.Int {
+func (k Keeper) GetTotalCollateral(ctx sdk.Context, denom string) sdk.Coin {
```

Comment:
How about renaming this to `GetCollateral` and `GetAllTotalCollateral` to `GetTotalCollateral`?

Reply:
After thinking more I think the current name is OK, but lmk what do you think

Reply:
Current functions that exist:
`GetAllTotalCollateral()`
`GetTotalCollateral(denom)`
`GetBorrowerCollateral(address)`
`GetCollateral(address,denom)`

`GetCollateral` already exists so we'd have to shuffle other things around. That can be saved for a refactor PR if we decide to do it.

Reply:
BTW, renaming is super easy in most of the editors - just few clicks.